docs/pages/pricing.tsx

```diff
@@ -18,7 +18,7 @@ export default function Pricing() {
   return (
     <BrandingProvider>
       <Head
-        title="Pricing - MUI"
+        title="MUI: Pricing"
```

Comment:
Are we sure about this change? I'm asking because:

1. On the docs page, we have `- MUI`

https://github.com/mui-org/material-ui/blob/54bcde1070ca17b466972ffc008d03637b69a3d8/docs/src/modules/components/AppLayoutDocs.js#L90

2. When you have multiple tabs open, you have a limited space to display. The logo already allows differentiating MUI tabs from the other.

Reply:
I'm for "Pricing - MUI"

Reply:
Ah, right. I guess `{title} - MUI` works fine too. I just had it changed to all pages so they are standardized. Only Pricing and About were following this pattern... But feel free to correct it!